autoreduce_frontend/templates/overview.html

```diff
@@ -17,17 +17,17 @@
                     {% endif %}
```

Comment:
Can you add `active` to the instrument `class`es (slightly above this, but it doesn't let me comment there)? That makes them a bit darker blue which I think looks nicer